toqito/states/tests/test_dicke.py

```diff
@@ -0,0 +1,50 @@
+"""Test dicke."""
+
+import numpy as np
+import pytest
+
+from toqito.states import dicke
+
+
+@pytest.mark.parametrize(
+    "num_qubit, num_excited, expected_state",
+    [
+        # Dicke state for 3 qubits and 1 excitation
+        (3, 1, np.array([0, 1/np.sqrt(3), 1/np.sqrt(3), 0, 1/np.sqrt(3), 0, 0, 0])),
+        # Dicke state for 4 qubits and 2 excitations (corrected expected state)
+        (4, 2, np.array([0, 0, 0, 1/np.sqrt(6), 0, 1/np.sqrt(6), 1/np.sqrt(6), 0,
+                         0, 1/np.sqrt(6), 1/np.sqrt(6), 0, 1/np.sqrt(6), 0, 0, 0])),
+    ],
+)
+def test_dicke_state(num_qubit, num_excited, expected_state):
+    """Test that dicke_state produces the correct vector state."""
+    np.testing.assert_array_almost_equal(dicke(num_qubit, num_excited), expected_state, decimal=6)
```

Comment:
we should not use `np.testing` as there's already an issue to move away from it.

Reply:
Good point, fixed in 0921559d.